l10n_br_fiscal/views/l10n_br_fiscal_action.xml

```diff
@@ -455,15 +455,38 @@
         </field>
     </record>
 
-    # Fiscal Document
-    <record id="document_action" model="ir.actions.act_window">
-        <field name="name">Document</field>
+    # Fiscal Document In
+    <record id="document_in_action" model="ir.actions.act_window">
+        <field name="name">Recebimento</field>
         <field name="type">ir.actions.act_window</field>
         <field name="res_model">l10n_br_fiscal.document</field>
         <field name="view_type">form</field>
         <field name="view_mode">kanban,tree,form</field>
         <field name="search_view_id" ref="document_search"/>
         <field name="view_id" ref="document_tree"/>
+        <field name="domain">['|', ('operation_type', '=', 'in'), ('operation_type', '=', 'all')]</field>
+        <field name="context">{'default_operation_type': 'in'}</field>
+        <field name="help" type="html">
+          <p class="o_view_nocontent_smiling_face">
+            Create a new Document
+          </p><p>
+            Odoo helps you easily track all activities
+            related to a fiscal operation.
```

Comment:
aqui fala de operação fiscal, mas o menu eh de documento fiscal
Tb acho que tem um erro de inglês, seria "helps you to easily track" faltou o to.